ontology_api/instance/instance_service.py

```diff
@@ -23,13 +23,14 @@ def add_instance(self, model_id, class_name: str, model_in: MinimalInstanceModel
 
         owl_class = onto[class_name]
         if owl_class is None:
+            onto.destroy()
             return InstanceModelOut(errors="Class named " + str(class_name) + " not found in Model " + str(model_id))
-
+        #t
```

Comment:
Does this comment have any meaning here? If so, it should be written in such a way that it can be understood by everyone....

Reply:
Fixed, the comment wasn't supposed to be there